tests/threshold/detection_filter-rule-flow/test.yaml

```diff
@@ -0,0 +1,36 @@
+requires:
+  min-version: 8
+
+pcap: ../threshold-rule-flow/icmp.pcap
+
+checks:
+  - filter:
+      count: 8
+      match:
+        event_type: alert
+        alert.signature_id: 1
+  - filter:
+      count: 7
```

Comment:
I do not understand why it is not 8, `count 1` should be a nop, right ?

Reply:
"The detection_filter keyword can be used to alert on every match after a threshold has been reached."
So matching starts after threshold is reached. The logic is not new btw, just also implemented for `by_flow`.